lib/pages/league/widgets/dialogs/rules_dialog.dart

```diff
@@ -0,0 +1,365 @@
+import 'package:ax_dapp/service/custom_styles.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:toggle_switch/toggle_switch.dart';
+
+const List<String> list = <String>['MLB', 'NFL', 'NBA'];
```

Comment:
Do we need this? I saw that you have an items list that is being used for the dropdown. If you do not need this, then please remove it.